modules/gapi/src/backends/ie/giebackend.cpp

```diff
@@ -973,22 +992,26 @@ struct InferROI: public cv::detail::KernelTag {
         GAPI_Assert(1u == uu.params.input_names.size());
         GAPI_Assert(2u == in_metas.size());
 
-        // 0th is ROI, 1st is input image
-        const auto &input_name = uu.params.input_names.at(0);
-        auto &&ii = uu.inputs.at(input_name);
-        auto &&mm = in_metas.at(1u);
-        configureInputInfo(ii, mm);
-        if (uu.params.layer_names_to_reshape.find(input_name) !=
-            uu.params.layer_names_to_reshape.end()) {
-            configureInputReshapeByImage(ii, mm, input_reshape_table);
-        }
-        ii->getPreProcess().setResizeAlgorithm(IE::RESIZE_BILINEAR);
+        // NB: Configuring input precision and network reshape must be done
+        // only for loadNetwork way.
+        if (uu.params.kind == cv::gapi::ie::detail::ParamDesc::Kind::Load) {
+            // 0th is ROI, 1st is input image
+            const auto &input_name = uu.params.input_names.at(0);
+            auto &&ii = uu.inputs.at(input_name);
```

Comment:
How does this work with imported network now? `IEUnit::InputsDataMap inputs` is empty, isn't it?

Reply:
Ok. I see if's condition)